core/src/mindustry/type/StatusEffect.java

```diff
@@ -21,6 +21,8 @@
     public float reloadMultiplier = 1f;
     /** Unit build speed multiplier. */
     public float buildSpeedMultiplier = 1f;
+    /** Unit drag multiplier */
```

Comment:
Javadocs should have punctuation, look at the surrounding javadocs.
```suggestion
    /** Unit drag multiplier. */
```